docs/src/modules/spring/pages/access-control.adoc

```diff
@@ -1 +1,36 @@
 = Access Control Lists (ACLs)
+
+This section describes functionality in Spring SDK related to ACLs (Access Control Lists), for information about how the ACLs are defined and control access see xref:spring:using-acls.adoc[Using ACLs]
```

Comment:
I would merge `Using ACLs` inside this page. We don't need to have two pages.

Reply:
done